### PR TITLE
Change README image to open use license

### PR DESCRIPTION
**Pull request by millengustavo**
Change to an open license image on README.

The image currently in use has no stated license to use.

I proposed an open license alternative from unsplash. Their license can be read in full [here](https://unsplash.com/license)

> Unsplash grants you an irrevocable, nonexclusive, worldwide copyright license to download, copy, modify, distribute, perform, and use photos from Unsplash for free, including for commercial purposes, without permission from or attributing the photographer or Unsplash. This license does not include the right to compile photos from Unsplash to replicate a similar or competing service.

Also reorganized the Table of Contents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,3 +315,7 @@ We already have email plugin: https://github.com/globocom/azkaban-jobtype-email
 ## Contribute
 
 For development and contributing, please follow [Contributing Guide](https://github.com/globocom/auror-core/blob/master/CONTRIBUTING.md) and ALWAYS respect the [Code of Conduct](https://github.com/globocom/auror-core/blob/master/CODE_OF_CONDUCT.md)
+
+## Image credits 
+
+<span>Photo by <a href="https://unsplash.com/@rhii?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Rhii Photography</a> on <a href="https://unsplash.com/?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>
```
Comment (thepabloaguilar) [inline README.md]: You can put this right after the image, seems more clear and understandable!

**Review by thepabloaguilar — CHANGES_REQUESTED** [review]
We have the same image in the documentation [here](https://github.com/globocom/auror-core/blob/master/docs/source/index.rst), you need to change both places!

File: README.md (hunk 1 above)
Comment (millengustavo) [reply README.md]: Done

**Review by thepabloaguilar — APPROVED** [review]
LGTM

**Pull request closed without merging**